packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -292,6 +310,9 @@ export class RecentsWidget extends Component {
       props.fetchSpace(sparkInstance, spaceId)
         .then((newSpace) => {
           if (newSpace) {
+            // Fetch latest notification features
+            props.getFeature('user', 'group-message-notifications', sparkInstance);
```

Comment:
You should use your constants that you created

Reply:
Do we have to fetch the user's features every time a new space loads?

Reply:
This is the only mercury event to my knowledge that it could use  to refresh a user's global notification (feature) updates.

Reply:
Ok gotcha, could you update the code comment with that info?

Reply:
I found another approach that's somewhat more elegant.